### PR TITLE
Added Notfound Page 

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Live Preview : https://dainty-strudel-f76e45.netlify.app/s

Files To Check 👍 

src/components/UI/NotFound/NotFound.jsx
src/components/UI/NotFound/notfound.css

### CODE REVIEW

File: src/components/UI/NotFound/NotFound.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import Navbar from '../Navbar/Navbar';
+import './notfound.css'
+const NotFound = () => {
+  return (
+    <>
+
```
Comment (gauravsinhaweb) [inline src/components/UI/NotFound/NotFound.jsx]: remove extra spacing here.

**Review by gauravsinhaweb — APPROVED** [review]
Rest code looks clean and good to me. Good to merge

File: src/components/UI/NotFound/NotFound.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const NotFound = () => {
+  return (
+    <>
+    
+    <Navbar/>
+    <section className='flex flex-column flex-justify-center flex-align-item-center'>
+      <img  className="page-404" src="https://res.cloudinary.com/dwhsfh3sc/image/upload/v1649259193/Offers-imgs/page404_gsf4ar.png" alt="Not-found-404" />
+       <h2>Oops.. page not found</h2>
+    </section>
+    </>
```
Comment (rahulyadav139) [inline src/components/UI/NotFound/NotFound.jsx]: Great work. Clean code.

**Review by rahulyadav139 — APPROVED** [review]

**Review by saurabhchirde — APPROVED** [review]
Clean code, Couldn't find any issue. Good to merge

**Pull request merged**